client/package.json

```diff
@@ -26,16 +27,15 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "lint": "eslint 'src/**/*.{js,jsx}'",
-    "format": "prettier --write \"src/**/*.{js,jsx,json,css}\"",
-    "format:check": "prettier --check 'src/**/*.{js,jsx,json,css}'",
-    "prepare": "cd .. && husky install client/.husky"
+    "lint": "eslint .",
+    "lint:fix": "eslint --cache --fix",
```

Comment:
Not sure if I'm running something wrong, but after running `yarn format` and `yarn lint:fix`, I'm still seeing the double quotes warning here:

![image](https://github.com/cpsc455-bugstorm/TravelersTea/assets/30674558/babbb043-0a66-414f-be19-1465d70f8aee)

Reply:
fixed it